custom_dc/build_repo.md

```diff
@@ -22,32 +22,26 @@ Then, restart Docker, specifying that repo as the argument to the `docker run` c
 
 If you have local changes that need to be built, or you are ready to deploy your site to Google Cloud, follow the procedures below.
 
-## One-time setup: download build dependencies {#download_deps}
-
-If you need to rebuild the repo locally, the `mixer` and `import` repos must be available as dependencies. Run this command one time to set the repos as subdirectories of the website repo:
-
-```shell  
-git submodule foreach git pull origin master  
-git submodule update --init --recursive  
-```
-
 ## Build the local repo
 
-From the `website` directory, sync to the latest version of the files in the repo:
+If you need to rebuild the repo locally, the `mixer` and `import` repos must be available as dependencies.
```

Comment:
mixer and import are submodules. So this can be updated to "the mixer and import submodules must also be updated".

Reply:
Done.